dist/Dockerfile.build/Dockerfile

```diff
@@ -8,18 +8,15 @@ RUN yum -y groupinstall 'Development Tools'
 RUN yum -y install openssl-devel
 
 ENV HOME="/root"
-
-# build: Rust stable toolchain
-RUN \
-  mkdir $HOME/rust && \
-  curl https://static.rust-lang.org/dist/rust-1.37.0-x86_64-unknown-linux-gnu.tar.gz | \
-  tar -xzvf - -C $HOME/rust --strip 1 && \
-  $HOME/rust/install.sh; \
-  rm -rf $HOME/rust
-
-# add cargo binaries to PATH
 ENV PATH="${HOME}/.cargo/bin:${PATH}"
 
+# build: Rust stable toolchain + 2 previous versions
+RUN mkdir $HOME/rust && \
+    curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.37.0 -y && \
+    rustup install 1.36.0 && \
+    rustup install 1.35.0 && \
+    rustup default 1.37.0
```

Comment:
do we need this even though `--default-toolchain` is defined above with this version? If this is required, I would favor to remove the version from the installation command.

Reply:
I left `--default-toolchain 1.37.0` there so that whenever stable changes we'd still get 1.37 installed - and we'd have to update to new stable with a PR.

It can be removed if we commit to supporting "rolling stable" though

Reply:
Ideally we would install the most recent stable, parse the version value and also install and test against the previous 2 or 3 versions. A solution which doesn't require any hardcoding, neither in the install section nor in the tests, would be awesome.

Reply:
Ah, got it, sure lets see if that can be nicely done